src/lib/yacc.rs

```diff
@@ -200,7 +224,10 @@ impl YaccParser {
                 assert!(s == 0);
                 Ok((i + e, self.src[i..i + e].to_string()))
             },
-            None         => Err(YaccError{kind: YaccErrorKind::IllegalName, off: i})
+            None         => {
+                let (line, col) = self.off_to_line_col(i);
```

Comment:
Might be worth it to refactor this in to a function `print_error` as you are repeating the same two lines 8 times.